pkg/api/create_person_handler.go

```diff
@@ -0,0 +1,49 @@
+package api
+
+import (
+	"encoding/json"
+
+	"github.com/darchlabs/api-example/pkg/person"
+	"github.com/go-playground/validator/v10"
+)
+
+type CreatePersonHandlerInputs struct {
```

Comment:
Maybe you must use the original `Person` type defined in the `Person` package because you can apply multiple metadata and you can centralize the person definition.

See the below photo:
 
<img width="683" alt="Screen Shot 2022-10-17 at 15 31 43" src="https://user-images.githubusercontent.com/8485620/196129822-eb3f773e-be00-4569-801d-fc4d699d54a7.png">

Reply:
Yeah, is better in that way. Done 👍